apps/vs-code-designer/src/app/commands/deploy/deploy.ts

```diff
@@ -332,41 +365,14 @@ async function getProjectPathToDeploy(
           tenant: `@appsetting('${workflowAppAADTenantId}')`,
           secret: `@appsetting('${workflowAppAADClientSecret}')`,
         };
-      } else {
-        managedConnection.authentication = {
-          type: 'ManagedServiceIdentity',
-        };
       }
+
       settingsToExclude.push(`${referenceKey}-connectionKey`);
     }
-
-    await writeFormattedJson(connectionsFilePath, parametizedConnections);
+    await writeFormattedJson(connectionsFilePathDeploy, parametizedConnections);
+    await writeFormattedJson(parametersFilePathDeploy, parametersJson);
 
     return deployProjectPath;
   }
   return undefined;
 }
-
-async function cleanAndRemoveDeployFolder(deployProjectPath: string): Promise<void> {
-  await fse.emptyDir(deployProjectPath);
-  fse.rmdirSync(deployProjectPath);
-}
-
-async function checkAADDetailsExistsInAppSettings(node: SlotTreeItem, identityWizardContext: IIdentityWizardContext): Promise<boolean> {
```

Comment:
nit: Is there a reason why this moved in the file? would be less churn if it stayed in place?

Reply:
it looks like there aren't any changes to the method itself

Reply:
@JoaquimMalcampo Thats right, if there aren't any changes on this, please avoid just moving code.

Reply:
reverted